deploy/kubernetes-1.13/pmem-csi.yaml

```diff
@@ -0,0 +1,256 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: csi-service-account
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: csi-cluster-role
+rules:
```

Comment:
I suspect that some of these RBAC entries are out-dated, for example the 

Did you copy-and-paste from previous deployments or from the canonical upstream definitions of the sidecar RBACs? Those are:
- https://github.com/kubernetes-csi/external-attacher/blob/v1.0.1/deploy/kubernetes/rbac.yaml
- https://github.com/kubernetes-csi/external-provisioner/blob/v1.0.1/deploy/kubernetes/rbac.yaml
- https://github.com/kubernetes-csi/cluster-driver-registrar/blob/v1.0.1/deploy/kubernetes/rbac.yaml

Please double-check the RBAC rules and explain in a comment from what and how they were assembled.

Reply:
> I suspect that some of these RBAC entries are out-dated, for example the CRD entries.

csi-cluster-driver-registar v1.0 is using CRDs to create CSIDriverCRD,  here is the code trace:

https://github.com/kubernetes-csi/cluster-driver-registrar/blob/ff1f9663191e085ae6201052e5ca9c85133664e7/cmd/cluster-driver-registrar/k8s_register.go#L64

Reply:
It shouldn't be doing that, and in master it was removed: https://github.com/kubernetes-csi/cluster-driver-registrar/commit/02b2bfe6bb0eea82bd1a2ab41544c1540c2b1637

I didn't know that v1.0.1 went out in that broken state (installing CRDs and not having the matching RBAC).

This is something that needs to go into our copy of the RBAC rules as a comment. We have to ensure that we remove it when updating to cluster-driver-registrar v1.1.0 or more generally, that we always keep our RBAC rules in sync with the sidecar releases that we use. That's why I was asking for a description that some other developer can use to create our copy from scratch.

Reply:
Added the comment in deployment files related to CRD rule